src/main/webapp/WEB-INF/tags/bm/include.jsp

```diff
@@ -25,7 +25,7 @@
 <%@ taglib prefix="uwfn" uri="http://my.wisc.edu/jsp/jstl/functions" %>
 <%@ taglib prefix="bm" tagdir="/WEB-INF/tags/bm" %>
     
-<%@ taglib prefix='portlet' uri='/WEB-INF/tags/portlet.tld' %>
+<%@ taglib prefix='portlet' uri='http://java.sun.com/portlet' %>

```

Comment:
Part of using the tld version in shared/lib.  Got it.  I suggest bumping to the 2.0 URI; http://java.sun.com/portlet_2_0